docs/notebooks/01_references.md

```diff
@@ -35,6 +35,8 @@ You can start by making a blank `KFactory` and add a single polygon to it.
 ```python
 import kfactory as kf
 
+kf.config.display_type = "image"
+
```

Comment:
Why change to image here? The docs will already run with this behavior. If you want to reproduces locally use `export KFACTORY_DISPLAY_TYPE="image"`